clang/lib/Sema/SemaTemplate.cpp

```diff
@@ -6165,6 +6254,18 @@ bool UnnamedLocalNoLinkageFinder::VisitHLSLAttributedResourceType(
   return Visit(T->getWrappedType());
 }
 
+bool UnnamedLocalNoLinkageFinder::VisitHLSLInlineSpirvType(
+    const HLSLInlineSpirvType *T) {
+  for (auto &Operand : T->getOperands()) {
+    if (Operand.isConstant() && Operand.isLiteral()) {
+      if (Visit(Operand.getResultType())) {
+        return true;
+      }
+    }
+  }
```

Comment:
nit: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
```suggestion
  for (auto &Operand : T->getOperands())
    if (Operand.isConstant() && Operand.isLiteral())
      if (Visit(Operand.getResultType()))
        return true;
```

Reply:
Done.